mods/ts/rules/bridges.yaml

```diff
@@ -0,0 +1,212 @@
+LOBRDG01:
+	Inherits: ^LowBridge
+	Building:
+		Footprint: _ _ _
+		Dimensions: 1, 3
+	LowBridge:
+
+LOBRDG02:
+	Inherits: ^LowBridge
+	Building:
+		Footprint: _ _ _
+		Dimensions: 1, 3
+	LowBridge:
+
+LOBRDG03:
+	Inherits: ^LowBridge
+	Building:
+		Footprint: _ _ _
+		Dimensions: 1, 3
+	LowBridge:
+
+LOBRDG04:
+	Inherits: ^LowBridge
+	Building:
+		Footprint: _ _ _
+		Dimensions: 1, 3
+	LowBridge:
+
+LOBRDG05:
+	Inherits: ^LowBridge
+	Building:
+		Footprint: _ _ _
+		Dimensions: 1, 3
+	LowBridge:
+
+LOBRDG06:
+	Inherits: ^LowBridge
+	Building:
+		Footprint: _ _ _
+		Dimensions: 1, 3
+	LowBridge:
+
+LOBRDG07:
+	Inherits: ^LowBridge
+	Building:
+		Footprint: _ _ _
+		Dimensions: 1, 3
+	LowBridge:
+
+LOBRDG08:
+	Inherits: ^LowBridge
+	Building:
+		Footprint: _ _ _
+		Dimensions: 1, 3
+	LowBridge:
+
+LOBRDG09:
+	Inherits: ^LowBridge
+	Building:
+		Footprint: _ _ _
+		Dimensions: 1, 3
+	LowBridge:
+
+LOBRDG10:
+	Inherits: ^LowBridge
+	Building:
+		Footprint: ___
+		Dimensions: 3, 1
+	LowBridge:
+
+LOBRDG11:
+	Inherits: ^LowBridge
+	Building:
+		Footprint: ___
+		Dimensions: 3, 1
+	LowBridge:
+
+LOBRDG12:
+	Inherits: ^LowBridge
+	Building:
+		Footprint: ___
+		Dimensions: 3, 1
+	LowBridge:
+
+LOBRDG13:
+	Inherits: ^LowBridge
+	Building:
+		Footprint: ___
+		Dimensions: 3, 1
+	LowBridge:
+
+LOBRDG14:
+	Inherits: ^LowBridge
+	Building:
+		Footprint: ___
+		Dimensions: 3, 1
+	LowBridge:
+
+LOBRDG15:
+	Inherits: ^LowBridge
+	Building:
+		Footprint: ___
+		Dimensions: 3, 1
+	LowBridge:
+
+LOBRDG16:
+	Inherits: ^LowBridge
+	Building:
+		Footprint: ___
+		Dimensions: 3, 1
+	LowBridge:
+
+LOBRDG17:
+	Inherits: ^LowBridge
+	Building:
+		Footprint: ___
+		Dimensions: 3, 1
+	LowBridge:
+
+LOBRDG18:
+	Inherits: ^LowBridge
+	Building:
+		Footprint: ___
+		Dimensions: 3, 1
+	LowBridge:
+
+LOBRDG19:
+	Inherits: ^LowBridge
+	Tooltip:
+		Name: Bridge Ramp
```

Comment:
Is it really necessary to distinguish these as ramps (as opposed to just "Bridge")?